src/app/configs/endpoints.ts

```diff
@@ -8,5 +8,6 @@ export const AUTH_ENDPOINTS = {
 };
 
 export const BOOK_ENDPOINTS = {
-  addBook: `${BASE_URL}/${SUFFIX}/book`,
+  addBook: `${BASE_URL}/${SUFFIX}/books`,
```

Comment:
The declared `addBook` endpoint does not exist in the back-end. The correct endpoint is `${BASE_URL}/${SUFFIX}/book`